.travis.yml

```diff
@@ -0,0 +1,22 @@
+language: python
+
+python:
+  - "2.7"
+
+before_install:
+  - "export DISPLAY=:99"
+  - "sh -e /etc/init.d/xvfb start"
+
+install:
+  - "pip install -r requirements.txt"
+  - "pip install -r test_requirements.txt"
+  - "pip install -r $VIRTUAL_ENV/src/xblock-sdk/requirements/base.txt"
```

Comment:
Is there a reason these requirements aren't just captured in the test_requirements.txt? similarly, is XBlock SDK not on pip? I would think you'd just install XBlock SDK and it would include whatever you need to develop an XBlock. That would include testing it, since testing is a part of development.